.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+# Description
+
+Please include a summary of the change or the fix you've made. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+## Type of change
+
+Please select atleast one
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+## Testing
+How was this change tested?
+
+## Documentation
+
+_Please list any documentation changes or additions. Please delete if not necessary._
+
+## Addtional Info
+
+_Please enter any additional information that could be helpful for the reviewers or just to document. Please delete if not necessary._
+
+## Checklist:
+
+- [ ] I have run swiftformat or other relevant tools to follow style guidelines.
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
+- [ ] New and existing unit tests pass locally with my changes
+- [ ] Any dependent changes have been merged and published 
+- [ ] I have attached screenshots for any UI changes. (Delete if not applicable)
```

Comment:
Not sure how valuable this is. `Workflow` or `WorkflowUI` do not have any UI components in the core libraries.

Reply:
I thought of this with regards to the samples i have been working on but you are right. This is not relevant outside of Samples so perhaps it doesn't need to be in the checklist.